src/main/java/org/cactoos/collection/CollectionNoNulls.java

```diff
@@ -82,9 +84,11 @@ public boolean contains(final Object item) {
         for (int idx = 0; idx < array.length; ++idx) {
             if (array[idx] == null) {
                 throw new IllegalStateException(
-                    String.format(
-                        "Item #%d of #toArray() is NULL", idx
-                    )
+                    new UncheckedText(
+                        new FormattedText(
+                            "Item #%d of #toArray() is NULL", idx
+                        )
+                    ).asString()
                 );
```

Comment:
@paulodamaso The error message contains an item index, that is not available in case of using `forEach`

Reply:
@proshin-roman How about using `AtomicInteger` for tracking the index?

Reply:
@paulodamaso TBH I don't think it's a good idea to make the code more complex without a good reason.

Reply:
@paulodamaso And again - refactoring this cycle is not a part of the ticket. The ticket says: use FormattedText instead of String.format everywhere, that's it.

Reply:
@proshin-roman I think you are right, i'm getting a bit out of scope here.